.github/workflows/fedora-39.yml

```diff
@@ -446,3 +447,81 @@ jobs:
           path: |
             *.json
             *.log
+
+  pre-iot-simplified-f39:
+    needs: pr-info
+    if: ${{ needs.pr-info.outputs.allowed_user == 'true' && github.event.issue.pull_request &&
+            (endsWith(github.event.comment.body, '/test-f39') ||
+            endsWith(github.event.comment.body, '/test-f39-simplified')) }}
+    runs-on: ubuntu-latest
+    env:
+      STATUS_NAME: iot-simplified-f39
+
+    steps:
+      - name: Create in-progress status
+        uses: octokit/request-action@v2.x
+        with:
+          route: 'POST /repos/${{ github.repository }}/statuses/${{ needs.pr-info.outputs.sha }}'
+          context: ${{ env.STATUS_NAME }}
+          state: pending
+          description: 'Runner has been deploying...'
+          target_url: 'https://github.com/${{ github.repository }}/actions/runs/${{ github.run_id }}'
+        env:
+          GITHUB_TOKEN: ${{ secrets.GITHUB_TOKEN }}
+
+  iot-simplified-f39:
+    needs: [pr-info, pre-iot-simplified-f39]
+    runs-on: [kite, x86_64, rhos-01, fedora-39, large]
+    env:
+      STATUS_NAME: iot-simplified-f39
+
+    steps:
+      - name: Create in-progress status
+        uses: octokit/request-action@v2.x
+        with:
+          route: 'POST /repos/${{ github.repository }}/statuses/${{ needs.pr-info.outputs.sha }}'
+          context: ${{ env.STATUS_NAME }}
+          state: pending
+          description: 'Test has been running...'
+          target_url: 'https://github.com/${{ github.repository }}/actions/runs/${{ github.run_id }}'
+        env:
+          GITHUB_TOKEN: ${{ secrets.GITHUB_TOKEN }}
+
+      - name: Clone repository
+        uses: actions/checkout@v3
+        with:
+          ref: ${{ needs.pr-info.outputs.sha }}
+          fetch-depth: 0
+
+      - name: run ostree-simplified-installer.sh
+        run: ./ostree-simplified-installer.sh
```

Comment:
You should define env var DOWNLOAD_NODE here.

Reply:
I thought fedora test does not require this env, for example, ostree.sh test does not have it, https://github.com/virt-s1/rhel-edge/blob/main/.github/workflows/fedora-39.yml#L93
Did I miss anything here?

Reply:
Oh, right. simplified installer test script does not need DOWNLOAD_NODE.